client/components/Recipes.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+// import Recipe from './Recipe'
+import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import Card from '@mui/material/Card'
```

Comment:
Your using a mix of different import methods for MUI ie named and default exports. Just stick with one